app/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/ {
+    chosen {
+        zephyr,console = &cdc_acm_uart;
```

Comment:
Is there not a built in console via a USB-UART IC available on the other USB port?

Reply:
Actually, there are three. It's the third one that was sending the logs, which is why I didn't find it at first.